src/components/Modals/OnboardingTransfer.vue

```diff
@@ -506,18 +483,6 @@ export default Vue.extend({
       };
       this.displayOptions.exam.push(exam);
     },
-    getCourseFromExam(type: 'AP' | 'IB', subject: string) {
```

Comment:
By deleting this function, what would replace the work it does? It seems like the current transfer page needs this function when the component is mounted, or I might need to change that for my pull request.

Reply:
What exactly does the function do and why does the transfer page need it? If it's to show the equivalent course(s), I would want to have a larger discussion about it. I removed it because the implementation isn't correct/consistent with the logic of `ExamCredit.ts`, and if there's a need for such a function I think it should go in that file.

Reply:
It gets the course an exam would give credit for and it's only useful for graying out courses in the dropdown if another entered exam would already give credit ex. If you entered AP Chem already then the chemistry IB test will be grayed out. I think it's fine to take it out then